stories/cli/try.fmf

```diff
@@ -127,3 +127,21 @@ link:
             package: vim
         execute:
             how: tmt
+
+/option:
+    story:
+        As a user I want an easy way to run common options in ``1minutetip`` and ``tmt run``
+
+    /epel:
+        story:
+            As a user I want an easy way to enable epel repository
+        description: |
+            Enable EPEL repository
+
+            .. versionadded:: 1.35
+
+        example:
+            - tmt try fedora@virtual --epel
+
+        link:
+          - implemented-by: /tmt/trying.py
```

Comment:
Would it make sense to point to plugins implementing these shortcuts? I would list them, knowing `prepare/feature` is the workhorse behind the option might be helpful, and an extra context.